data/src/main/java/com/architect/coders/mu8/data/remote/mapper/ComicMapper.kt

```diff
@@ -0,0 +1,14 @@
+package com.architect.coders.mu8.data.remote.mapper
+
+import com.architect.coders.mu8.data.remote.model.getallComic.ComicResponse
+import com.architect.codes.mu8.model.Comic
+
+class ComicMapper {
+    fun transform(comicR: ComicResponse): Comic {
+        return Comic(
+            id = comicR.id,
+            title = comicR.title?: "",
+            thumbnailUrl = comicR.thumbnail.path + "." + comicR.thumbnail.extension
```

Comment:
Implement this as Kotlin suggest using the `${}`.

Reply:
done